Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Next
 
+### Changed
+- Add uploadData(Data) case to Task enum. [#2042](https://github.com/Moya/Moya/pull/2042) by [@julienbodet](https://github.com/julienbodet).
+
 ### Changed
 - Moya's Swift version is now Swift 5.1. [#1931](https://github.com/Moya/Moya/pull/1931) by [@BasThomas](https://github.com/BasThomas).
```

Comment:
There is already a section for changed, can you move it to here?

Reply:
Sure!